apps/rehabstod/src/pages/CareProvider/components/CareProviderAccordion.tsx

```diff
@@ -18,31 +18,34 @@ export function CareProviderAccordion({
 }) {
   return (
     <details className="group">
-      <summary className="ml-5 flex cursor-pointer items-center space-x-2">
-        <span className="group-open:rotate-270 inline-block h-3 w-3 origin-center rotate-90">
-          <IDSIcon name="chevron" width="100%" height="100%" className="h-full w-full" />
-        </span>
-        <div className="flex w-full justify-between">
-          <label
-            htmlFor={unit.id}
-            className={`flex cursor-pointer items-center ${selectedRadio === unit.id ? 'font-bold' : ''} w-full justify-between`}>
-            <span>{unit.namn}</span>
-            <Radio
+      <summary className="flex cursor-pointer items-start space-x-2">
+        <div className="flex w-full items-center justify-between">
+          <IDSRadio>
+            <input
+              type="radio"
               name="selectedUnit"
               value={unit.namn}
               id={unit.id}
-              onChange={(event) => handleChooseUnit(event, provider, unit)}
               onClick={(e) => {
                 const detailsEl = e.currentTarget.closest('details')
                 if (detailsEl) {
                   detailsEl.setAttribute('open', '')
                 }
               }}
+              onChange={(event) => handleChooseUnit(event, provider, unit)}
             />
-          </label>
+            <label
+              htmlFor={unit.id}
+              className={`flex cursor-pointer items-center ${selectedRadio === unit.id ? 'font-bold' : ''} mb-0 w-full justify-between`}>
+              <span>{unit.namn}</span>
+            </label>
+          </IDSRadio>
+          <span className="group-open:rotate-270 inline-block h-3 w-3 origin-center rotate-90">
```

Comment:
Let's replace this `group-open:rotate-270` with `group-open:-rotate-90` and remove our 270 extension from the `tailwind.config.cjs`

Reply:
I changed that in the test branch, but i forgot to remove it from extensions!

Reply:
Alright, nice! 😄